bokeh/models/sources.py

```diff
@@ -325,14 +325,13 @@ def patch(self, patches, setter=None):
 
         self.data._patch(self.document, self, patches, setter)
 
-class GeoJSONDataSource(ColumnDataSource):
+class GeoJSONDataSource(DataSource):
```

Comment:
Class structure must be the same as in bokehjs, so you have to introduce `ColumnarDataSource` and set bases appropriately.